### PR TITLE
Update ticket-input.hbs

### PR DESCRIPTION
**Pull request by khushishikhu**
#4475 #4482

### CODE REVIEW

**Comment by iamareebjamal** [comment]
This does not fix the issue. Shotgun debugging can't help in this case

**Comment by khushishikhu** [comment]
sir till number-10 it is showing correct validation but when number increases after 10 it doesn't

**Comment by khushishikhu** [comment]
sir this validation is working when min and max value are both greater than 10. but not when either of the min or max is a one-digit number.

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5885738&bid=19266012)

File: app/components/forms/wizard/basic-details-step.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ export default Component.extend(FormMixin, EventWizardMixin, {
 
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
-    $.fn.form.settings.rules.checkMaxMinPrice = () => {
+    $.fn.form.settings.rules.checkMaxMinPrice = function(min_price,max_price){
```
Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:136]: Codacy found an issue: [A space is required after ','.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5885738)

Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:136]: Codacy found an issue: [Missing space before opening brace.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5885738)

Comment (niranjan94) [inline app/components/forms/wizard/basic-details-step.js:136]: Codacy found an issue: ['min_price' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5885738)

**Comment by khushishikhu** [comment]
sir please review it. and guide me what i need to do.

**Comment by iamareebjamal** [comment]
Unfortunately, I don't know what is to be done in this bug. I'll have to debug it and find out, at that point, I might as well go ahead and fix it myself. Debugging is a part of bug solving. First, you need to reproduce the issue, then you need to find out why is it happening and then solve it.

**Pull request closed without merging**